src/rpc/net.cpp

```diff
@@ -130,6 +125,11 @@ static RPCHelpMan getpeerinfo()
                     {RPCResult::Type::BOOL, "inbound", "Inbound (true) or Outbound (false)"},
                     {RPCResult::Type::BOOL, "bip152_hb_to", "Whether we selected peer as (compact blocks) high-bandwidth peer"},
                     {RPCResult::Type::BOOL, "bip152_hb_from", "Whether peer selected us as (compact blocks) high-bandwidth peer"},
+                    {RPCResult::Type::STR_HEX, "services", /*optional=*/true, "The services offered"},
```

Comment:
I'm not sure if changing these fields to optional requires a release note, since missing fields that used to be always present could break clients that assume they'll be there. See #25176 for example.

Reply:
I changed this back to be non-optional just to stick with the previous behavior (i.e. when the peer has not sent a `version` message yet we default to  `NODE_NONE`).

Reply:
:+1: Seems pragmatic